src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1194,7 +1192,7 @@ private String readStdOut(Process p) throws IOException, InterruptedException {
         String line;
         StringBuffer allLines = new StringBuffer();
         while ((line = in.readLine())!= null) {
-            allLines.append(line);
+            allLines.append(line + " ");
```

Comment:
We are using StringBuffer to avoid using concatenation. Could you make this .append(line).append(" ")?